inference-engine/src/mkldnn_plugin/mkldnn_graph_dumper.cpp

```diff
@@ -119,22 +119,34 @@ InferenceEngine::CNNNetwork dump_graph_as_ie_ngraph_net(const MKLDNNGraph &graph
     ngraph::ParameterVector params;
     ngraph::NodeVector to_hold;
 
+    bool shouldDumpConstantNodes = true;
+
+#ifdef CPU_DEBUG_CAPS
+    shouldDumpConstantNodes = !(graph.getProperty().debugCaps.shouldDumpConstNodes == "NO");
+#endif
+
     auto get_inputs = [&] (const MKLDNNNodePtr & node) {
         auto pr_edges = node->getParentEdges();
```

Comment:
could you please replace "auto" with "const auto&" for this lambda where it is applicable.